bindings_ffi/src/mls.rs

```diff
@@ -1208,21 +1192,38 @@ impl FfiConversations {
             _ => None,
         };
 
-        let convo = if account_identities.is_empty() {
-            let group = self
-                .inner_client
-                .create_group(group_permissions, metadata_options)?;
-            group.sync().await?;
-            group
-        } else {
-            self.inner_client
-                .create_group_with_members(&account_identities, group_permissions, metadata_options)
-                .await?
-        };
+        let convo = self
+            .inner_client
+            .create_group(group_permissions, metadata_options)?;
 
         Ok(Arc::new(convo.into()))
     }
 
+    pub async fn create_group(
+        &self,
+        account_identities: Vec<FfiIdentifier>,
+        opts: FfiCreateGroupOptions,
+    ) -> Result<Arc<FfiConversation>, GenericError> {
+        log::info!(
+            "creating group with account addresses: {}",
+            account_identities
+                .iter()
+                .map(|ident| format!("{ident}"))
+                .collect::<Vec<_>>()
+                .join(", ")
+        );
+
+        let convo = self.create_group_optimistic(opts).await?;
+
+        if !account_identities.is_empty() {
+            convo.add_members(account_identities).await?;
+        };
+
+        convo.sync().await?;
```

Comment:
I don't think we need to call `sync()` here, if and only if `add_members` was called, because that method already syncs